localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -469,11 +470,11 @@ void EKFLocalizer::measurementUpdatePose(const geometry_msgs::msg::PoseWithCovar
     ekf_.getXelement(delay_step * dim_x_ + IDX::Y), ekf_yaw;
```

Comment:
Why not we apply the same modification as line l.546 - 548?

Reply:
Updated in [83b63a3](https://github.com/autowarefoundation/autoware.universe/pull/2698/commits/83b63a35dc73dce635a8f757b01cd5a6db01ca13)